src/lib/cache-control.spec.ts

```diff
@@ -0,0 +1,54 @@
+import * as assert from 'node:assert/strict';
+import { describe, it } from 'node:test';
+import { CacheControl } from './cache-control.js';
```

Comment:
Leave an empty line above this import statement to indicate local vs. global imports.